share/picongpu/examples/LaserWakefield/include/picongpu/simulation_defines/param/laser.param

```diff
@@ -21,6 +21,20 @@
 
 #pragma once
 
+// unitless
+#ifndef UI_MACRO_LASER_A0
+#define UI_MACRO_LASER_A0 1.5
```

Comment:
minor: we nowadays try to align `#if`s [like this](https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#7-preprocessor):
```C++
#ifndef UI_MACRO_LASER_A0
#   define UI_MACRO_LASER_A0 1.5
#endif
```